srcpkgs/pmacct/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'pmacct'
+pkgname=pmacct
+version=1.7.6
+revision=1
+build_style=gnu-configure
+hostmakedepends="libpcap-devel autoconf automake libtool"
```

Comment:
Why does it need `autoconf` and `automake`, given that it's using the `gnu-configure` build style, so at least `configure` has already been generated?

Reply:
> Why does it need `autoconf` and `automake`, given that it's using the `gnu-configure` build style, so at least `configure` has already been generated?

Because the makefile under src/external_libs calls autogen.sh for libcdada among other things, and generally expects to generate everything for libcdada. Indeed this shouldn't be necessary as `configure` is already generated for it, but I didn't want to mess with the makefile right now, this needs to be properly fixed in the upstream project.